fastlane/Fastfile

```diff
@@ -20,7 +20,7 @@ platform :ios do
   lane :unit_tests do
     # add actions here: https://docs.fastlane.tools/actions
     test_args = {
-        scheme: "TrustylibUnitTests",
+        scheme: "Trustylib",
         device: "iPhone 11",
```

Comment:
Unit tests are linked to the `TrustylibUnitTests` xcode schme, setting the scheme to `Trustylib` may not execute  the unit tests.

Can you check please if the unit tests are running with `bundle exec fastlane unit_tests` command?

Reply:
That is correct 🙂  the unit tests still remain linked to `TrustylibUnitTests`.
The change of scheme here fixes an issue with 0 percent coverage on Sonar and coverage on test files on codecov.
Now Sonar will show the correct coverage percent and codecov shows the generated coverage by tests for `Trustylib` scheme.

> Can you check please if the unit tests are running with bundle exec fastlane unit_tests command?

Yes 🙂 The unit tests are running with `bundle exec fastlane unit_tests` command:
https://app.circleci.com/pipelines/github/superus8r/etrusted-ios-trustbadge-library/25/workflows/61191f03-9aa4-49ec-9bc7-3275eb2147c2/jobs/43

Reply:
Awesome! thanks for confirming